### PR TITLE
Create joyshockmapper.json

### PR DESCRIPTION
**Pull request by derspyy**

### CODE REVIEW

**Comment by derspyy** [comment]
btw to clarify i added this here instead of scoop-extras bc betterjoy is here

File: bucket/joyshockmapper.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "homepage": "https://github.com/Electronicks/JoyShockMapper",
+    "description": "A tool for PC gamers to play games with DualShock 4s, JoyCons, and Pro Controllers. Gyro aiming, flick stick.",
+    "license": "MIT",
+    "version": "v3.3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/v3.3.0/JoyShockMapper_x64.zip",
+            "hash": "1B3C3B5C1B1D2B09075221AE283D171820ED5198021C8DB6417CAA6BD1994DCE",
```
Comment (Calinou) [inline bucket/joyshockmapper.json:9]: Use lowercase for hashes:

```suggestion
            "hash": "1b3c3b5c1b1d2b09075221ae283d171820ed5198021c8db6417caa6bd1994dce",
```

File: bucket/joyshockmapper.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            "bin": [["JoyShockMapper_x64\\JoyShockMapper.exe", "jsm"]],
+            "extract_dir": "JoyShockMapper_x64"
+        },
+        "32bit": {
+            "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/v3.3.0/JoyShockMapper_Win32.zip",
+            "hash": "AAD648244BCC7201A8EA86209CF54AF7A558017B237CBB630E7B63A9370E5DCC",
```
Comment (Calinou) [inline bucket/joyshockmapper.json:15]:
```suggestion
            "hash": "aad648244bcc7201a8ea86209cf54af7a558017b237cbb630e7b63a9370e5dcc",
```

**Review by Calinou — APPROVED** [review]
Thanks!

File: bucket/joyshockmapper.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "homepage": "https://github.com/Electronicks/JoyShockMapper",
+    "description": "A tool for PC gamers to play games with DualShock 4s, JoyCons, and Pro Controllers. Gyro aiming, flick stick.",
+    "license": "MIT",
+    "version": "v3.3.0",
```
Comment (Calinou) [inline bucket/joyshockmapper.json:5]:
```suggestion
    "version": "3.3.0",
```

File: bucket/joyshockmapper.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    ],
+    "checkver": {
+        "github": "https://github.com/Electronicks/JoyShockMapper",
+        "regex": "download/(?<build>v[\\d\\.]+[a-z]?)",
+        "replace": "${1}"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/$matchBuild/JoyShockMapper_x64.zip"
```
Comment (Calinou) [inline bucket/joyshockmapper.json:34]:
```suggestion
                "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/v$version/JoyShockMapper_x64.zip"
```

File: bucket/joyshockmapper.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/$matchBuild/JoyShockMapper_Win32.zip"
```
Comment (Calinou) [inline bucket/joyshockmapper.json:37]:
```suggestion
                "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/v$version/JoyShockMapper_Win32.zip"
```

File: bucket/joyshockmapper.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
+{
+    "homepage": "https://github.com/Electronicks/JoyShockMapper",
+    "description": "A tool for PC gamers to play games with DualShock 4s, JoyCons, and Pro Controllers. Gyro aiming, flick stick.",
+    "license": "MIT",
+    "version": "3.3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/v3.3.0/JoyShockMapper_x64.zip",
+            "hash": "1b3c3b5c1b1d2b09075221ae283d171820ed5198021c8db6417caa6bd1994dce",
+            "bin": [["JoyShockMapper_x64\\JoyShockMapper.exe", "jsm"]],
```
Comment (Calinou) [inline bucket/joyshockmapper.json:10]:
```suggestion
```

File: bucket/joyshockmapper.json [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+            "extract_dir": "JoyShockMapper_x64"
+        },
+        "32bit": {
+            "url": "https://github.com/Electronicks/JoyShockMapper/releases/download/v3.3.0/JoyShockMapper_Win32.zip",
+            "hash": "aad648244bcc7201a8ea86209cf54af7a558017b237cbb630e7b63a9370e5dcc",
+            "bin": [["JoyShockMapper_Win32\\JoyShockMapper.exe", "jsm"]],
```
Comment (Calinou) [inline bucket/joyshockmapper.json:16]:
```suggestion
```

File: bucket/joyshockmapper.json [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+            "extract_dir": "JoyShockMapper_Win32"
+        }
+    },
+    "persist": [
```
Comment (Calinou) [inline bucket/joyshockmapper.json:20]:
```suggestion
    },
    "bin": "JoyShockMapper.exe",
    "persist": [
```

**Comment by Calinou** [comment]
I tested this manifest locally and I'm getting this when starting Joyshock Mapper, either through a command prompt or using the Start Menu:

image

**Comment by derspyy** [comment]
it seems to be an issue with the /current/ junction link that scoop does, i'll contact the main dev

**Comment by derspyy** [comment]
btw i'm kinda new to making scoop stuff so thanks for the help !

**Comment by derspyy** [comment]
oops sory

**Comment by Calinou** [comment]
This PR appears to have fallen apart (and I keep getting notifications due to CI force-pushing updates), so I'll close it for now.

Feel free to open another PR with the manifest if you wish :slightly_smiling_face:

**Pull request closed without merging**